Gems/Profiler/Code/Source/ImGuiCpuProfiler.cpp

```diff
@@ -24,13 +24,17 @@
 #include <AzCore/std/sort.h>
 #include <AzCore/std/string/conversions.h>
 #include <AzCore/std/time.h>
-#include <AzCore/Time/ITime.h>
 
 namespace Profiler
 {
     constexpr AZStd::sys_time_t ProfilerViewEdgePadding = 5000;
     constexpr size_t InitialCpuTimingStatsAllocation = 8;
 
+    constexpr int MinSavableFrameCount = 30; // 1 second @ 30 fps
+    constexpr int MaxSavableFrameCount = 2000;
+
+    constexpr int MaxUpdateFrequencyMs = 2000; // 2 seconds
+
```

Comment:
Would it be worth exposing these as cvars as well?

Reply:
The settings registry might be more appropriate for these.  There are a few others in the gem that likely should be converted as well.